launcher/ui/instanceview/VisualGroup.cpp

```diff
@@ -321,7 +227,7 @@ int VisualGroup::contentHeight() const
 
 int VisualGroup::numRows() const
 {
-    return rows.size();
+    return (int)rows.size();
```

Comment:
What's this cast for?

Reply:
warning suppression  on data loss from conversion form a larger type like size_t I suspect

Reply:
Qt uses int

Reply:
![image](https://github.com/PrismLauncher/PrismLauncher/assets/31988415/af288dc2-f873-4f0c-ade9-814cacd81f72)

Reply:
Ah, i'm using Qt 5 for breeze lol